CHANGELOG.md

```diff
@@ -12,12 +23,12 @@
 * The `beadledom-correlation` module.
 
 ### Changed
-* Updated resteasy to 4.x. Following changes are affected if your service is also uplifting to use 4.x version of Resteasy.
-  * Make sure to install the [Beadledom JAX-RS CorrelationIdFilter](jaxrs/src/main/java/com/cerner/beadledom/jaxrs/provider/CorrelationIdFilter.java) on the server side so that correlationIds are forwarded properly when consuming a beadledom client.
+* Updated resteasy to 3.11. Following changes are affected if your service is also uplifting to use 4.x version of Resteasy.
```

Comment:
This should still be resteasy 4.x correct?

Reply:
no. In 3.6 we uplifted to 3.11.

Reply:
yeah but in 4.x branch, we uplifted to resteasy4.x so the changelog for beadledom-4 should say that we uplifted to resteasy to 4.x correct?

Reply:
Correct, Under `4.0 - In Development` I have
`Upgraded RESTEasy to 4.5.5.Final.`

Is there something else that needs to change?

Reply:
I think thats all we need. Thanks!